project/plugins.sbt

```diff
@@ -1 +1,3 @@
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.2.0-M8")
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.4")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
```

Comment:
Use sbt-assembly to build a fat jar (to be compiled into a binary with Graal's native-image)